client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/operations/OperationsFactory.java

```diff
@@ -81,10 +81,10 @@ public RemoveIfUnmodifiedOperation newRemoveIfUnmodifiedOperation(byte[] key, lo
    }
 
    public ReplaceIfUnmodifiedOperation newReplaceIfUnmodifiedOperation(byte[] key,
-            byte[] value, int lifespanSeconds, int maxIdleTimeSeconds, long version) {
+            byte[] value, int lifespanSeconds, int lifespanNanos, int maxIdleSeconds, int maxIdleNanos, long version) {
```

Comment:
Hmmm, wouldn't it neater to have `int lifespan, TimeUnit lifespanUnit, int maxIdle, TimeUnit maxIdleUnit' even if we only support two units?